### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/rustdesk/actions/runs/3876807141#summary-10533316862

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/rustdesk/actions/runs/3876807271#summary-10533317071

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ Please ensure that you are running these commands from the root of the RustDesk
 - **[src/rendezvous_mediator.rs](https://github.com/rustdesk/rustdesk/tree/master/src/rendezvous_mediator.rs)**: Communicate with [rustdesk-server](https://github.com/rustdesk/rustdesk-server), wait for remote direct (TCP hole punching) or relayed connection
 - **[src/platform](https://github.com/rustdesk/rustdesk/tree/master/src/platform)**: platform specific code
 - **[flutter](https://github.com/rustdesk/rustdesk/tree/master/flutter)**: Flutter code for mobile
-- **[flutter/web/js](https://github.com/rustdesk/rustdesk/tree/master/flutter/web/js)**: Javascript for Flutter web client
+- **[flutter/web/js](https://github.com/rustdesk/rustdesk/tree/master/flutter/web/js)**: JavaScript for Flutter web client
```
Comment (jsoref) [inline README.md:201]: Brand

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ efforts from contributors on the same issue.
 
 - Add tests relevant to the fixed bug or new feature.
 
-For specific git instructions, see [GitHub workflow 101](https://github.com/servo/servo/wiki/Github-workflow).
+For specific git instructions, see [GitHub workflow 101](https://github.com/servo/servo/wiki/GitHub-workflow).
```
Comment (jsoref) [inline docs/CONTRIBUTING.md:38]: URLs in the wiki aren't case sensitive

File: fastlane/metadata/android/en-US/full_description.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@ An open-source remote desktop application, the open source TeamViewer alternativ
 Source code: https://github.com/rustdesk/rustdesk
 Doc: https://rustdesk.com/docs/en/manual/mobile/
 
-In order for a remote device to control your Android device via mouse or touch, you need to allow RustDesk to use the "Accessibility" service, RustDesk uses AccessibilityService API to implement Addroid remote control.
+In order for a remote device to control your Android device via mouse or touch, you need to allow RustDesk to use the "Accessibility" service, RustDesk uses AccessibilityService API to implement Android remote control.
```
Comment (jsoref) [inline fastlane/metadata/android/en-US/full_description.txt:5]: Hopefully (this correction wasn't automatic...)

File: flutter/lib/common/widgets/login.dart [hunk 4]
```diff
@@ -538,7 +538,7 @@ Future<bool?> loginDialog() async {
           ),
           LoginWidgetOP(
             ops: [
-              ConfigOP(op: 'Github', iconWidth: 20),
+              ConfigOP(op: 'GitHub', iconWidth: 20),
```
Comment (jsoref) [inline flutter/lib/common/widgets/login.dart:541]: I can't tell if this is dangerous

File: flutter/lib/desktop/widgets/tabbar_widget.dart [hunk 5]
```diff
@@ -906,7 +906,7 @@ class _TabState extends State<_Tab> with RestorationMixin {
                       children: [
                         _buildTabContent(),
                         Obx((() => _CloseButton(
-                              visiable: hover.value && widget.closable,
+                              visible: hover.value && widget.closable,
```
Comment (jsoref) [inline flutter/lib/desktop/widgets/tabbar_widget.dart:909]: Scary/Confusing

File: flutter/linux/CMakeLists.txt [hunk 6]
```diff
@@ -9,7 +9,7 @@ set(BINARY_NAME "rustdesk")
 # https://wiki.gnome.org/HowDoI/ChooseApplicationID
 set(APPLICATION_ID "com.carriez.flutter_hbb")
 
-# Explicitly opt in to modern CMake behaviors to avoid warnings with recent
+# Explicitly opt into modern CMake behaviors to avoid warnings with recent
```
Comment (jsoref) [inline flutter/linux/CMakeLists.txt:12]: Changes for `opt-in` / `into` have been controversial

File: libs/clipboard/src/lib.rs [hunk 7]
```diff
@@ -21,7 +21,7 @@ pub use context_send::*;
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(tag = "t", content = "c")]
-pub enum ClipbaordFile {
+pub enum ClipboardFile {
```
Comment (jsoref) [inline libs/clipboard/src/lib.rs:24]: Not sure how "public" this is...

File: libs/clipboard/src/windows/wf_cliprdr.c [hunk 8]
```diff
@@ -795,11 +795,11 @@ static HRESULT STDMETHODCALLTYPE CliprdrDataObject_QueryGetData(IDataObject *Thi
 }
 
 static HRESULT STDMETHODCALLTYPE CliprdrDataObject_GetCanonicalFormatEtc(IDataObject *This,
-																		 FORMATETC *pformatectIn,
+																		 FORMATETC *pformatetcIn,
```
Comment (jsoref) [inline libs/clipboard/src/windows/wf_cliprdr.c:798]: I believe this should be `etc` and not `ect`, but I'm not terribly confident.

File: src/server/input_service.rs [hunk 9]
```diff
@@ -487,7 +487,7 @@ fn active_mouse_(conn: i32) -> bool {
         return false;
     }
 
-    let in_actived_dist = |a: i32, b: i32| -> bool { (a - b).abs() < MOUSE_ACTIVE_DISTANCE };
+    let in_active_dist = |a: i32, b: i32| -> bool { (a - b).abs() < MOUSE_ACTIVE_DISTANCE };
```
Comment (jsoref) [inline src/server/input_service.rs:490]: I'm not sure about this

File: src/ui/cm.tis [hunk 10]
```diff
@@ -31,7 +31,7 @@ class Body: Reactor.Component
         var disconnected = c.disconnected;
         var show_elevation_btn = handler.can_elevate() && show_elevation && !c.is_file_transfer && c.port_forward.length == 0;
         var show_accept_btn = handler.get_option('approve-mode') != 'password';
-       // below size:* is work around for Linux, it already set in css, but not work, shit sciter
+       // below size:* is a workaround for Linux, it already set in css, but not work, shit sciter
```
Comment (jsoref) [inline src/ui/cm.tis:34]: notable

**Review by jsoref — COMMENTED** [review]
Most changes automatically suggested by Google Sheets. All fault mine.

**Comment by rustdesk** [comment]
All look good, Thank you so much.

**Pull request merged**